depthcharge/data/hdf5.py

```diff
@@ -106,6 +107,15 @@ def _reindex(self):
 
         self._file_offsets = np.cumsum([0] + offsets)
 
+        # Build a map of 1D indices to 2D locations:
+        grp_idx = 0
+        for lin_idx in range(offsets[-1]):
+            if lin_idx >= offsets[grp_idx]:
```

Comment:
**nitpick:** `grp_idx += lin_idx >= offsets[grp_idx]` ?

Reply:
haha thanks - I'll change it 😄